lib/components/AssociatedGenesTable.js

```diff
@@ -0,0 +1,173 @@
+import React, { Component, Fragment } from 'react';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+
+import OtTable from './OtTable';
+
+const getColumnsAll = schemas => {
+  const columns = [
+    { id: 'gene', label: 'Gene' },
+    {
+      id: 'overallScore',
+      label: 'Overall G2V',
+      renderCell: rowData => {
+        return rowData.overallScore.toPrecision(3);
+      },
+    },
+  ];
+  schemas.forEach(schema => {
+    columns.push({
+      id: schema.sourceId,
+      label: schema.sourceId,
+      renderCell: rowData => {
+        return rowData[schema.sourceId]
+          ? rowData[schema.sourceId].toPrecision(3)
+          : 'No information';
+      },
+    });
+  });
+  return columns;
+};
+
+const getDataAll = genesForVariant => {
+  const data = [];
+  genesForVariant.forEach(item => {
+    const row = { gene: item.gene.symbol, overallScore: item.overallScore };
+    item.qtls.forEach(qtl => {
+      row[qtl.sourceId] = qtl.aggregatedScore;
+    });
+    item.intervals.forEach(interval => {
+      row[interval.sourceId] = interval.aggregatedScore;
+    });
+    item.functionalPredictions.forEach(fp => {
+      row[fp.sourceId] = fp.aggregatedScore;
+    });
+    data.push(row);
+  });
+  return data;
+};
+
+const getTissueColumns = (schemas, sourceId) => {
+  const columns = [{ id: 'gene', label: 'Gene' }];
+  schemas
+    .find(schema => schema.sourceId === sourceId)
+    .tissues.forEach(tissue => {
+      columns.push({
+        id: tissue.id,
+        label: tissue.name,
+        renderCell: rowData => {
+          return rowData[tissue.id]
+            ? rowData[tissue.id].toPrecision(3)
+            : 'No information';
+        },
+      });
+    });
+  return columns;
+};
+
+const getTissueData = (genesForVariantSchema, genesForVariant, sourceId) => {
+  const data = [];
+  let searchField;
+
+  genesForVariantSchema.qtls.forEach(qtl => {
+    if (qtl.sourceId === sourceId) {
+      searchField = 'qtls';
+    }
+  });
+
+  !searchField &&
+    genesForVariantSchema.intervals.forEach(interval => {
+      if (interval.sourceId === sourceId) {
+        searchField = 'intervals';
+      }
+    });
+
+  !searchField &&
+    genesForVariantSchema.functionalPredictions.forEach(fp => {
+      if (fp.sourceId === sourceId) {
+        searchField = 'functionalPredictions';
+      }
+    });
+
+  genesForVariant.forEach(geneForVariant => {
+    const row = { gene: geneForVariant.gene.symbol };
+    const element = geneForVariant[searchField].find(
+      item => item.sourceId === sourceId
+    );
+
+    if (element) {
+      element.tissues.forEach(elementTissue => {
+        row[elementTissue.tissue.id] =
+          elementTissue.score || elementTissue.beta;
+      });
+    }
+
+    data.push(row);
+  });
+  return data;
+};
+
+class AssociatedGenesTable extends Component {
+  state = {
+    value: 0,
```

Comment:
Below you use `value === schema.sourceId` when iterating the tissue tabs, so perhaps it'd be better to use an appropriate string constant for the *All* tab instead of `0`? Such as:
```
const OVERVIEW = 'overview';
...
state: {
  value: OVERVIEW
}
...
{value === OVERVIEW && <OtTable columns={columnsAll} data={dataAll} />}
```